HW2_1_Students_Teachers_groops/HW2_1_Students_Teachers_groops/Program.cs

```diff
@@ -0,0 +1,192 @@
+﻿using HW2_1_Students_Teachers_groops.Classes;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HW2_1_Students_Teachers_groops
+{
+    class Program
+    {
+        public static int ReadNumberForMenu(int itemOfMenu)
+        {
+            int number = 0;
+            while (!int.TryParse(Console.ReadLine(), out number) || number < 0 || number > itemOfMenu)
+            {
+                PrntErrMes();
+            }
+            return number;
+        }
+
+        public static void PrntErrMes()
+        {
+            Console.WriteLine("Wrong format!! Try again...");
+        }
+
+        public static void PrntErrMes1()
+        {
+            Console.WriteLine("No free teacheres!!");
+        }
+        public static void PrntErrMes2(EnmTeacherRole numGroup, Teacher.TechersGroup a)
```

Comment:
The name of the method does not express the idea of the method
Please review chose names